### ISSUE TITLE
Empty "()" parenthesis will break up for long lines

### ISSUE THREAD

**Issue by chasegiunta** [trivago/prettier-plugin-twig-melody#10]
Take the following code-snippet

```
{{ craft.someCoolObject.someMethodToUse({ foo: 'bar', bar: 'baz', baz: 'foo' }).all() }}
```
will format to:
```
{{
  craft.someCoolObject.someMethodToUse({
    foo: 'bar',
    bar: 'baz',
    baz: 'foo'
  }).all(

  )
}}
```

**Comment by twbartel** [comment]
Hi @chasegiunta, I just published v0.1.1, which contains a bug fix for this issue. Give it a try when you have time.

**Comment by twbartel** [comment]
Closing for now.
